pkg/karmadactl/util/idempotency.go

```diff
@@ -48,6 +48,8 @@ func CreateOrUpdateSecret(client kubeclient.Interface, secret *corev1.Secret) er
 			return fmt.Errorf("unable to update Secret: %v", err)
 		}
 	}
+	klog.V(2).Infof("Secret %s/%s has been created or updated.", secret.Namespace, secret.Name)
+
```

Comment:
Have you tested `karmadactl init`? I wonder what this log looks like from CLI.

Reply:
```sh
root@karmada-dev-linux-jiangwei [08:09:01 PM] [~/.kube] 
-> # /root/workspace/tmp/_output/bin/linux/amd64/kubectl-karmada register 172.18.0.3:32443 --token lm6cdu.lcm4wafod2jmjvty --discovery-token-ca-cert-hash sha256:9bf5aa53d2716fd9b5568c85db9461de6429ba50ef7ade217f55275d89e955e4 --cluster-name=member --v=3
I1121 20:09:03.415570 2138892 register.go:298] registering cluster. cluster name: member
I1121 20:09:03.415603 2138892 register.go:299] registering cluster. cluster namespace: karmada-cluster
[preflight] Running pre-flight checks
I1121 20:09:03.415622 2138892 register.go:411] validating the existence of file /etc/karmada/bootstrap-karmada-agent.conf
I1121 20:09:03.415634 2138892 register.go:411] validating the existence of file /etc/karmada/karmada-agent.conf
I1121 20:09:03.415643 2138892 register.go:411] validating the existence of file /etc/karmada/pki/ca.crt
[prefligt] All pre-flight checks were passed
[karmada-agent-start] Waiting to perform the TLS Bootstrap
I1121 20:09:03.422369 2138892 register.go:754] [discovery] Created cluster-info discovery client, requesting info from "172.18.0.3:32443"
I1121 20:09:03.427314 2138892 register.go:792] [discovery] Requesting info from "172.18.0.3:32443" again to validate TLS against the pinned public key
I1121 20:09:03.432151 2138892 register.go:809] [discovery] Cluster info signature and contents are valid and TLS certificate validates against pinned roots, will use API Server "172.18.0.3:32443"
I1121 20:09:03.432167 2138892 register.go:426] [discovery] Using provided TLSBootstrapToken as authentication credentials for the join process
I1121 20:09:03.432179 2138892 register.go:437] [discovery] writing bootstrap karmada-agent config file at /etc/karmada/bootstrap-karmada-agent.conf
I1121 20:09:03.432574 2138892 register.go:446] [discovery] writing CA certificate at /etc/karmada/pki/ca.crt
[karmada-agent-start] Waiting to construct karmada-agent kubeconfig
I1121 20:09:03.572593 2138892 register.go:500] waiting for the client certificate to be issued
I1121 20:09:04.574822 2138892 register.go:508] signing certificate successfully
I1121 20:09:04.574858 2138892 register.go:532] writing bootstrap karmada-agent config file at /etc/karmada/karmada-agent.conf
[karmada-agent-start] Waiting the necessary secret and RBAC
I1121 20:09:04.585592 2138892 idempotency.go:51] Secret karmada-system/karmada-kubeconfig has been created or updated.
I1121 20:09:04.591420 2138892 rbac.go:183] ClusterRole karmada-agent has been created or updated.
I1121 20:09:04.598572 2138892 rbac.go:206] ClusterRoleBinding karmada-agent has been created or updated.
[karmada-agent-start] Waiting karmada-agent Deployment
I1121 20:09:07.614601 2138892 check.go:49] pod: karmada-agent-7487b6f48d-tbdnm is ready. status: Running

cluster(member) is joined successfully
```